### DISCUSSION TITLE
Unable to verify academic affiliation with my student card

### DISCUSSION THREAD

**Discussion by madgehugo** [community/community#73130] [GitHub Education]
### Hi everyone,

Yesterday I attempted several times to verify my academic status in order to get the GitHub Pro discount. I used my student card, which contains all the requested information, but I kept getting the following issues:

- The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
- The image you selected does not appear to contain your school name. Ensure your image (photo or upload) is legible. We do not accept school logos. If you are taking a photo of a screen try to have it as large a resolution as possible. You may include multiple documents in your image. If your original document is not in English then you might take a photo of the original next to an English translation.
- You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

For issues 1 and 3, I cannot understand where this problem comes from as I am literally in my university's campus while doing this submission. I have tried applying with and without the university's VPN and neither worked.
For issue 2, the student card clearly states the school's name. While it is also the university's logo, it is a logo that has no extra elements in it other than the name (ETH Zurich).

Has anyone else had this issue? How can i resolve this?

Thanks!
- Hugo

**Comment by williamregal** [comment]
The [HP HPE0-S60  exact questions answer](https://www.myexamcollection.com/HPE0-S60-vce-questions.htm), also known as the "Architecting Multi-Site HPE Storage Solutions" exam, is a vital certification that equips IT professionals with the knowledge and skills necessary to design and implement complex storage solutions using HPE technologies. This certification is highly regarded in the industry, making it a valuable asset for individuals aiming to advance their careers in the field of IT.

**Comment by williamregal** [comment]
[HP HPE0-S60  exact questions answer](https://www.myexamcollection.com/HPE0-S60-vce-questions.htm) is a badge of excellence that showcases your proficiency in architecting multi-site HPE storage solutions. It is recognized globally, setting you apart as an expert in this domain. Achieving this certification is a testament to your commitment to professional growth and development.
